front/types/assistant/conversation.ts

```diff
@@ -37,7 +37,7 @@ export type AssistantUserMessageType = {
   sId: string;
   visibility: AssistantMessageVisibility;
   version: number;
-  parentMessageId: string;
+  parentMessageId: string | null;
```

Comment:
Do you confirm this can be `null` @spolu  ?

Reply:
Yes user messages always have null parent Ids I think. We can remove it from the type altogether?

Reply:
For now ? But I expect you'll be able to reply as a user to a message of another user ?

Reply:
Fair, but we'll have to add more infrastructure around that anyway?

Reply:
Yeah ok, let's remove then 👍

Reply:
got rid of it